omnibus/lib/symbols_inspectors.rb

```diff
@@ -0,0 +1,70 @@
+require 'find'
+require 'singleton'
+
+class ForbiddenSymbolsFoundError < StandardError
+
+end
+
+# Helper class to locate `dumpbin.exe` on Windows
+class Dumpbin
+  include Singleton
+
+  def path
+    if !@dumpbin
+      vsroot = ENV["VCINSTALLDIR"]
+      if !vsroot
+        print("VC not detected in environment; checking other locations")
+        # VSTUDIO_ROOT is a Datadog env var, it should be present
+        vsroot = ENV["VSTUDIO_ROOT"]
+        if !vsroot
+          raise RuntimeError.new("Could not find a Visual Studio installation")
+        end
+        # VSTUDIO_ROOT is not defined with "VC" in its path, so append it
+        vsroot += "\\VC\\"
+      end
+      Find.find(vsroot) do |path|
+        if File.basename(path) == "dumpbin.exe"
+          @dumpbin = path
+          Find.prune
+        else
+          next
+        end
+      end
+      if !@dumpbin
+        raise RuntimeError.new("Could not find dumpbin.exe")
+      end
+    end
+    @dumpbin
+  end
+
+  def call(binary_path)
+    `"#{path()}" /SYMBOLS "#{binary_path}"`.encode("UTF-8", invalid: :replace)
```

Comment:
Interesting that there is no explicit space on left and right from "/SYMBOLS".  I presume that ruby adds it implicitly (otherwise it would not work)